templates/governing-board/elections.html

```diff
@@ -0,0 +1,118 @@
+{% extends "page.html" %}
+{% import "macros/nominee.html" as nominee %}
+{% block head_extra %}
+<script>
+    window.addEventListener('DOMContentLoaded', () => {
+        const nomineeSections = document.querySelectorAll('.nominees');
+        for (const nomineeSection of nomineeSections) {
+            const nomineesDetails = nomineeSection.querySelectorAll('details');
+
+            nomineeSection.querySelector('.expand-all')?.addEventListener('click', () => {
+                for (const nomineeDetail of nomineesDetails) {
+                    nomineeDetail.setAttribute("open", "");
+                    console.log(nomineeDetail);
+                }
+            });
+
+            nomineeSection.querySelector('.collapse-all')?.addEventListener('click', () => {
+                for (const nomineeDetail of nomineesDetails) {
+                    nomineeDetail.removeAttribute("open");
+                    console.log(nomineeDetail);
+                }
+            });
+        }
+    });
+</script>
+{% endblock %}
+{% block content %}
+<article id="elections" class="content">
+    <header>
+        <h1>{{ page.title }}</h1>
+    </header>
+
+    {{ page.content | safe }}
+
+    {% set nominees_path = page.path ~ "nominees.toml" %}
+    {% set nominees = load_data(path=nominees_path, format="toml") %}
+
+    {% if nominees.platinum_members %}
```

Comment:
It might make sense to have a short (a sentence or 2 max) description of what each category means/who  they are for

Reply:
Ah good catch, I've added that in the latest commit